mysql-test/main/regression_function.test

```diff
@@ -0,0 +1,126 @@
+#
+# Various tests for regression functions
+#
+
+--disable_warnings
+DROP TABLE IF EXISTS t1, t2;
```

Comment:
This line isn't necessary. Assume test cases always start with a clean environment.